web/annotation_editor_params.js

```diff
@@ -62,6 +63,11 @@ class AnnotationEditorParams {
     editorFreeHighlightThickness.addEventListener("input", function () {
       dispatchEvent("HIGHLIGHT_THICKNESS", this.valueAsNumber);
     });
+    editorHighlightShowAll.addEventListener("click", function () {
+      const checked = this.getAttribute("aria-pressed") === "true";
+      this.setAttribute("aria-pressed", !checked);
```

Comment:
Shouldn't this attribute instead be updated via the "annotationeditorparamschanged" listener below, similar to all the other buttons, since currently this feels inconsistent?